chain-extensions/token/src/lib.rs

```diff
@@ -228,7 +228,7 @@ where
 
 	<orml_currencies::Pallet<T> as MultiCurrency<T::AccountId>>::transfer(
 		currency_id,
-		env.ext().caller(),
+		env.ext().caller().account_id()?,
```

Comment:
I am not sure what we where doing before, but what error handling you have in mind? Right now the extension call would just fail.
About the likeliness, I have no idea how it could fail since it is something coming from the call itself.

Reply:
What exactly is the return value of `account_id()`? I didn't find it in online documentation.

Reply:
It returns ` Result<&T::AccountId, DispatchError>`. Yeah, I had to go back to the IDE.

Reply:
Okay, well if it's a `DispatchError` then I guess it's fine if we just keep the `?`.